mapping/mapping-csv-files.md

```diff
@@ -4,12 +4,12 @@ You can download as CSV files the mapping information between Income Tax (Making
+| [sa106_mapping_v2.csv](csv/sa106_mapping_v2.csv) | [Foreign (GOV.UK)](https://www.gov.uk/government/publications/self-assessment-foreign-sa106) | [Individual Losses](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individual-losses-api) <br />[Individuals Dividends Income](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-dividends-income-api) [Individuals Foreign Income](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-foreign-income-api) <br />[Individuals Insurance Policies Income](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-insurance-policies-income-api) <br />[Individuals Other Income](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-other-income-api) <br />[Individuals Pensions Income](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-pensions-income-api) <br />[Individuals Reliefs](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-reliefs-api) <br />[Individuals Savings Income](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-savings-income-api) <br />[Property Business](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/property-business-api)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
Include the `<br />` tag as currently both are on the same line & reformat table